tests/testthat/test-eval-select.R

```diff
@@ -81,6 +112,8 @@ test_that("predicate outputs are type-checked", {
 })
 
 test_that("eval_select() produces correct backtraces", {
+  skip_if(interactive()) # backtraces look different
```

Comment:
This is going to make it hard to regenerate / monitor backtraces. What exactly was different?

Reply:
In ESS `devtools::test()` runs in the same interactive session. I'll take a look to make the test stable.

Reply:
If I remove from this PR, is it good to merge?

Reply:
yes thanks!